backend/spec/workers/archive_hubee_portail_enrollments_worker_spec.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe ArchiveHubeePortailEnrollmentsWorker, type: :worker do
+  subject { described_class.new }
+
+  describe "#perform" do
+    context "When there are duplicated hubee_portail enrollments" do
+      before do
+        create(:enrollment, :hubee_portail, :validated, organization_kind: :clamart)
+        @duplicated_enrollment_1 = create(:enrollment, :hubee_portail, :submitted, organization_kind: :clamart)
+        @duplicated_enrollment_2 = create(:enrollment, :hubee_portail, :draft, organization_kind: :clamart)
+        @email_content = File.open(Rails.root.join("app/views/enrollment_mailer/hubee_portail/archive.text.erb")) { |f| f.readline }.chomp
+      end
+
+      it "archive enrollments" do
+        result = subject.perform
+        expect(result.pluck(:id, :status)).to eq([[@duplicated_enrollment_1.id, "archived"], [@duplicated_enrollment_2.id, "archived"]])
```

Comment:
No need to use `pluck` here. Pluck is an activeRecord request that we don't need to do here. It takes times to execute and can slow down the execution time of this tests suite. 

you can check with 2 expects 
`expect(result[0].status).to eq("archived")`
`expect(result[1].status).to eq("archived")`

Same, you do not need to store the 2 enrollments in a variable like `@duplicated_enrollment_1` this way because you will be working directly on the index of the array `result`

Reply:
you do not check that the `event` `archive` is created. 
for this you can simply add : 

```ruby
it "creates archive events" do
  result = subject.perform
  
  expect(result[0].events.last.name).to eq("archive")
```